File: C/Assignment-2/Prog10.c
#include<stdio.h>


int fibo(int a, int b, int n)
{
    if (n<1)
        return a;
    else 
        return fibo(b, a+b, n-1);
}
int main(){
    int N = 0;
    scanf("%d", &N);
    for (int i = 0; i < N; i++)
        printf("%d, ", fibo(0,1,i));
    return 0;
}

File: Python/Assignment-2/Prog10.py
def fibo(a:int, b:int, n:int) -> int:
    if n==0:
        return a
    else:
        return fibo(b, a+b, n-1)

upper, lower = map(int, input("Enter the upper range and the lower range : ").split())
for i in range(upper,lower+1):
    print(fibo(0, 1, i), end = " ")
